config/helm/crd_bootstrap_cainjection.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Patch for setting our `cert-manager.io/inject-ca-from` annotation (overriding namespace part) in all bootstrap CRDs (label selector is used in kustomization.yaml to apply this to all bootstrap CRDs).